include/chibi/sexp.h

```diff
@@ -890,13 +890,13 @@ SEXP_API int sexp_idp(sexp x);
 #define _or_integer_flonump(x)
 #endif
 
+#define sexp_make_integer(ctx, x) sexp_make_fixnum(x)
+#define sexp_make_unsigned_integer(ctx, x) sexp_make_fixnum(x)
 #if SEXP_USE_BIGNUMS
-SEXP_API sexp sexp_make_integer(sexp ctx, sexp_lsint_t x);
-SEXP_API sexp sexp_make_unsigned_integer(sexp ctx, sexp_luint_t x);
```

Comment:
These are public APIs used by the FFI so need to be preserved (the sexp_make_fixnum definition is not compatible).  It's ok to ifdef though, only using sexp_l[su]int_t when these are native integer types, and fall back to long and unsigned long otherwise.

Reply:
Mmmh I'm not really confortable with all the API/ABI implications of what I've done, so please massage it in any way you see fit :)